xamarin/Xamarin.Forms
# [Android] Fix SwipeView touch issue scrolling on Device

### Issues Resolved ### 

Fix SwipeView issue intercepting (sometimes) touch scrolling in CollectionView on a device (only happened on device).

- fixes #10313 
- fixes #10875 

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

<img width="478" alt="issue10875" src="https://user-images.githubusercontent.com/6755973/83256195-490c1280-a1b2-11ea-84e7-a057cf98818d.png">


### Testing Procedure ###
Launch Core Gallery in a device and navigate to the SwipeView samples. Select the "SwipeView Vertical CollectionView" sample and verify that everything works fine (can scroll, open and close the Swipe, tap a SwipeItem, etc).

### PR Checklist ###
